README.md

```diff
@@ -49,6 +49,7 @@ Additions/corrections are welcome! Please check out [Contributing Guidelines](CO
 | [Render](http://render-conf.com/) | Oxford, United Kingdom | March 23, 2018 | January 8, 2018 | Yes (plus £250 honorarium) | [Yes](http://2018.render-conf.com/code-of-conduct) |
 | [VueConf US](http://www.vueconf.us/) | New Orleans, Louisiana | March 26 - 28, 2018 | December 4, 2017 | Yes | [Yes](http://www.vueconf.us/codeofconduct) |
 | [#PerfMatters US](https://perfmattersconf.com/) | Redwood City, California | March 27 - 28, 2018 | October 1, 2017 | Yes | [Yes](https://perfmattersconf.com/code/) |
+| [CSSConf CN](https://css.w3ctech.com/?l=en) | XAmoy(Xiamen), China | March 31, 2018 | March 8, 2018 | Yes | [Unspecified](https://perfmattersconf.com/code/) |
```

Comment:
Unspecified shouldn't be linked. This looks like a copy and paste mistake?

Reply:
Yes, you are absolutely right. I am so sorry